test/e2e/cluster.go

```diff
@@ -27,36 +27,31 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/ready"
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 	"github.com/Azure/ARO-RP/pkg/util/version"
-	"github.com/Azure/ARO-RP/test/util/project"
 )
 
 const (
 	testPVCName = "e2e-test-claim"
 )
 
 var _ = Describe("Cluster", Serial, func() {
-	var p project.Project
+	var project Project
 
 	BeforeEach(func(ctx context.Context) {
 		By("creating a test namespace")
 		testNamespace := fmt.Sprintf("test-e2e-%d", GinkgoParallelProcess())
-		p = project.NewProject(clients.Kubernetes, clients.Project, testNamespace)
-		err := p.Create(ctx)
-		Expect(err).NotTo(HaveOccurred(), "Failed to create test namespace")
+		project = BuildNewProject(ctx, clients.Kubernetes, clients.Project, testNamespace)
 
 		By("verifying the namespace is ready")
 		Eventually(func(ctx context.Context) error {
-			return p.Verify(ctx)
+			return project.VerifyProjectIsReady(ctx)
```

Comment:
Do we need to keep the `Eventually` in the test itself wherever `VerifyProjectIsReady()` is used? I ask because there is already an `Eventually` with timeout in the `CreateK8sObjectWithRetry` function, which `VerifyProjectIsReady()` calls.

Reply:
Resolving as I now understand this commit to be part of https://github.com/Azure/ARO-RP/pull/3366 :)